VideoLocker/src/androidTest/java/org/edx/mobile/test/feature/interactor/LandingScreenInteractor.java

```diff
@@ -21,6 +21,11 @@ public LandingScreenInteractor observeLandingScreen() {
         return this;
     }
 
+    public RegistrationScreenInteractor navigateToRegistrationScreen() {
```

Comment:
Since usage of `registration` word is now much greater than `signUp`, it'll be more plausible to just refactor `onSignUpView()` to `onRegistrationView()` instead.